x-pack/plugins/cases/server/services/user_actions/builders/comment.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ActionTypes, Actions } from '../../../../common/api';
+import { UserActionBuilder } from '../abstract_builder';
+import { UserActionParameters, BuilderReturnValue } from '../types';
+
+export class CommentUserActionBuilder extends UserActionBuilder {
+  build(args: UserActionParameters<'comment'>): BuilderReturnValue {
+    return this.buildCommonUserAction({
+      ...args,
+      action: args.action ?? Actions.update,
```

Comment:
I'm probably missing the context here but should we just make `args.action` required?

Reply:
Because all builders share the same arguments (they only differ on the payload) enforcing the action to be required will make the consumer of the API need to select an action even if it is not being used by the builder.